diff --git a/anykernel.sh b/anykernel.sh
index 7906b22..744e051 100755
--- a/anykernel.sh
+++ b/anykernel.sh
@@ -4,18 +4,18 @@
 ## AnyKernel setup
 # begin properties
 properties() { '
-kernel.string=ExampleKernel by osm0sis @ xda-developers
+kernel.string=daoudeddy @ Github
 do.devicecheck=1
 do.modules=0
 do.systemless=1
 do.cleanup=1
 do.cleanuponabort=0
-device.name1=maguro
-device.name2=toro
-device.name3=toroplus
-device.name4=tuna
+device.name1=surya
+device.name2=
+device.name3=
+device.name4=
 device.name5=
-supported.versions=
+supported.versions=11
 supported.patchlevels=
 '; } # end properties
 
